README.md

```diff
@@ -5,98 +5,119 @@
+## Open Source
+API Blueprint is completely open sourced under the MIT license.
+Any [contribution][contribute] is highly appreciated.
 
-        Hello World!
-```
+[contribute]: #Contribute
 
-in your favourite Markdown editor. With this API description in your API GitHub repository you can discuss it with others and iterate on your API Design.
+## At home on GitHub
+API Blueprint language is recognized by GitHub. You can
+[search for API Blueprint][search] or use the `apib` language identifier for
+[syntax highlighting][gfm].
 
-To learn more about the API Blueprint syntax jump directly to the [API Blueprint Tutorial](Tutorial.md) or take a look at some [examples](https://github.com/apiaryio/api-blueprint/tree/master/examples).
+[search]: https://github.com/search?utf8=%E2%9C%93&q=language%3A%22API+Blueprint%22&type=Repositories&ref=advsearch&l=API+Blueprint&l=
 
-## Complete Lifecycle
-Describing your API is only the start. The API Blueprint can be used by variety of tools from an interactive documentation, SDK generator, debugging proxy and mock server to API testing and analytics tools.
+[gfm]: https://help.github.com/articles/github-flavored-markdown/#syntax-highlighting
 
-![API Blueprint Lifecycle](assets/lifecycle.png)
+## Getting started
+All it takes to describe an endpoint of your API is to write:
 
-Visit the [tooling section](http://apiblueprint.org/#tooling) of the API Blueprint website to find more about the tools you can use with API Blueprint.
+```apib
+# GET /message
++ Response 200 (text/plain)
 
-## Machines
-Building tools for API Blueprint is possible thanks to its machine-friendly face:
+        Hello World!
+```
 
-```json
-{
-    "_version": "2.0",
-    "metadata": [],
-    "name": "",
-    "description": "",
+in your favorite plain text editor.
 
-    ...
-}
-```
+With this blueprint you can already get a [mock][], [documentation][] and
+[test][] for your API before you even start coding.
 
-\[[full listing](https://github.com/apiaryio/api-blueprint-ast#json-serialization)\]
+To learn more about the API Blueprint syntax jump directly to the
+[API Blueprint Tutorial][tutorial] or take a look at some [examples][].
 
-It is the task for the API Blueprint parser – [Drafter][] or one of its language bindings to translate the API Blueprint Markdown representation into a machine friendly format – AST.
+[mock]: http://docs.apibstart.apiary.io/#reference/0/message/get?console=1
+[documentation]: http://docs.apibstart.apiary.io
+[test]: https://github.com/apiaryio/dredd
+[tutorial]: Tutorial.md
+[examples]: https://github.com/apiaryio/api-blueprint/tree/master/examples
 
-If you are interested in building tools for API Blueprint or just to integrate it with your workflow check out the [Developing tools for API Blueprint](https://github.com/apiaryio/api-blueprint/wiki/Developing-tools-for-API-Blueprint).
+## Version
+- Current version: [1A9](https://github.com/apiaryio/api-blueprint/releases/tag/format-1A9)
```

Comment:
Off-topic, should we have released 1A10 with MSON nullability? Or would MSON features not warrant API Blueprint updates?